eras/dijkstra/impl/testlib/Test/Cardano/Ledger/Dijkstra/Arbitrary.hs

```diff
@@ -65,7 +68,7 @@ instance Arbitrary (TxBody SubTx DijkstraEra) where
       <*> arbitrary
       <*> arbitrary
 
-instance Arbitrary (TxBody TopTx DijkstraEra) where
+instance HasOKey TxId (Tx SubTx DijkstraEra) => Arbitrary (TxBody TopTx DijkstraEra) where
```

Comment:
This should not be needed, since this is a fully saturated constraint.
Because this constraint is fully saturated it means this instance does not exist and thus is not satisfiable.
```suggestion
instance Arbitrary (TxBody TopTx DijkstraEra) where
```

Please add this instance to `Cardano.Ledger.Core`:
```haskell
instance EraTxBody era => HasOKey TxId (TxBody l era) where
  toOKey = txIdTxBody
```